src/components/dropdown-filter-ajax/dropdown-filter-ajax.js

```diff
@@ -207,26 +225,39 @@ class DropdownFilterAjax extends DropdownFilter {
    *
    * @method getData
    * @param {String} query The search term
-   * @param {Object} page The page number to get
    */
-  getData = (query = '', page = 1) => {
+  getData = (query = '') => {
```

Comment:
After checking a bit more in depth on it - we need to keep `page` param on `getData` because it's being called all over the place and that's the real value of page, not what's in the state it would seem. Check here: 
https://github.com/Sage/carbon/blob/595298425833d81f6f0d47a92e86ab6271f26471/src/components/dropdown-filter-ajax/dropdown-filter-ajax.js#L169
https://github.com/Sage/carbon/blob/595298425833d81f6f0d47a92e86ab6271f26471/src/components/dropdown-filter-ajax/dropdown-filter-ajax.js#L217

So rather than pick the page value later on in `getParams`, we need to pass it down form `getData` call.